modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,174 @@
+
+.. Get the name of the node where the buidah container is running:
++
+[source,terminal]
+----
+$ oc get pod -o wide
+----
+
+.. Run a debugging pod on the node where the `buildah` pod is running:
++
+[source,terminal]
+----
+$ oc debug nodes/<node-name>
+----
+
+.. Change the root directory for the current running processes:
++
+[source,terminal]
+----
+$ chroot /host
+----
+
+.. List the user namespaces:
++
+[source,terminal]
+----
+$ lsns -t user
+----
+
+[NOTE]
+====
+* Although {pipelines-shortname} runs your `buildah` container as a root user, CRI-O runs the container as a user in the host namespace.
+
+* When you run a command in a user namespace, your container runs as root (user id 0) but has user privileges on the host. {product-title} uses CRI-O as its container runtime interface. CRI-O has a workload feature to run a pod in the user namespace.
+====
+
+.Example: Modified Buildah task
+[source,yaml]
+----
+...
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: buildah-userns
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: image-build
+spec:
+  description: >-
+    Buildah task builds source into a container image and
+    then pushes it to a container registry.
+
+    Buildah Task builds source into a container image using Project Atomic's
+    Buildah build tool.It uses Buildah's support for building from Dockerfiles,
+    using its buildah bud command.This command executes the directives in the
+    Dockerfile to assemble a container image, then pushes that image to a
+    container registry.
+
+  params:
+  - name: IMAGE
+    description: Reference of the image buildah will produce.
+  - name: BUILDER_IMAGE
+    description: The location of the buildah builder image.
+    default: registry.redhat.io/rhel8/buildah@sha256:99cae35f40c7ec050fed3765b2b27e0b8bbea2aa2da7c16408e2ca13c60ff8ee
+  - name: STORAGE_DRIVER
+    description: Set buildah storage driver
+    default: vfs
+  - name: DOCKERFILE
+    description: Path to the Dockerfile to build.
+    default: ./Dockerfile
+  - name: CONTEXT
+    description: Path to the directory to use as context.
+    default: .
+  - name: TLSVERIFY
+    description: Verify the TLS on the registry endpoint (for push/pull to a non-TLS registry)
+    default: "true"
+  - name: FORMAT
+    description: The format of the built container, oci or docker
+    default: "oci"
+  - name: BUILD_EXTRA_ARGS
+    description: Extra parameters passed for the build command when building images.
+    default: ""
+  - name: PUSH_EXTRA_ARGS
+    description: Extra parameters passed for the push command when pushing images.
+    type: string
+    default: ""
+  workspaces:
+  - name: source
+
+  results:
+  - name: IMAGE_DIGEST
+    description: Digest of the image just built.
+
+  steps:
+  - name: build
+    securityContext:
+      runAsUser: 1000
```

Comment:
task in 1.8.0 has completely different `securityContext`, only the other doc mentions `runAsUser: 1000`

Reply:
I will apply these patches in a separate task: https://issues.redhat.com/browse/RHDEVDOCS-4657
For the time being, I am in favor of closing the issue so that at least docs till 1.7 are present.

Reply:
Based on discussion in the thread https://coreos.slack.com/archives/CG5GV6CJD/p1667818845838809, the content will be completely rewritten, and therefore these comments are not useful anymore.